src/widget.cpp

```diff
@@ -18,6 +18,7 @@
 #include "core/geometry_func.hpp"
 #include "settings_type.h"
 #include "querystring_gui.h"
+#include "core/random_func.hpp"
```

Comment:
I'm not seeing what random functionality is being used here. I reckon you had a more colourful version before this incarnation, right?

Reply:
Correct indeed, I shall fix this. Always-white turned out to be easier to see...